loglist/loglist.go

```diff
@@ -61,6 +72,40 @@ type STH struct {
 	TreeHeadSignature []byte `json:"tree_head_signature"`
 }
 
+// NewFromJSON creates a LogList from JSON encoded data.
+func NewFromJSON(llData []byte) (*LogList, error) {
+	var ll LogList
+	if err := json.Unmarshal(llData, &ll); err != nil {
+		return nil, fmt.Errorf("failed to parse log list: %v", err)
+	}
+	return &ll, nil
+}
+
+// NewFromSignedJSON creates a LogList from JSON encoded data, checking a
+// signature along the way. The signature format should be ?????
```

Comment:
????? -> TODO?

Reply:
Oops, fixed -- I forgot to come back and update that after I figured out the signature format